README.md

```diff
@@ -1,20 +1,195 @@
-[![Build Status](https://travis-ci.org/xm-online/tmf-ms-customer.svg?branch=master)](https://travis-ci.org/xm-online/tmf-ms-customer) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?&metric=sqale_index&branch=master&project=xm-online:tmf-ms-customer)](https://sonarcloud.io/dashboard/index/xm-online:tmf-ms-customer) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?&metric=ncloc&branch=master&project=xm-online:tmf-ms-customer)](https://sonarcloud.io/dashboard/index/xm-online:tmf-ms-customer) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?&metric=coverage&branch=master&project=xm-online:tmf-ms-customer)](https://sonarcloud.io/dashboard/index/xm-online:tmf-ms-customer)
+# tmf-ms-customer
 
-# customer
-
-This application was generated using JHipster 6.5.1, you can find documentation and help at [https://www.jhipster.tech/documentation-archive/v6.5.1](https://www.jhipster.tech/documentation-archive/v6.5.1).
+This application was generated using JHipster 7.8.1, you can find documentation and help at [https://www.jhipster.tech](https://www.jhipster.tech).
 
 This is a "microservice" application intended to be part of a microservice architecture, please refer to the [Doing microservices with JHipster][] page of the documentation for more information.
 
-This application is configured for Service Discovery and Configuration with Consul. On launch, it will refuse to start if it is not able to connect to Consul at [http://localhost:8500](http://localhost:8500). For more information, read our documentation on [Service Discovery and Configuration with Consul][].
+## How to use this template
+1. Clone xm-ms-template to separate folder.
+2. Start project in you local XM environment and check if it is working, run tests.
+3. Remove `.git/` folder before project modification.
+4. Add project to new git repository where it will evolve and live.
+5. Find and replace `tmf-ms-customer` term everywhere in the projects.
+6. Find end remove everything related to `ExampleEntityFirst` and `ExampleEntitySecond` (after checking how it is proposed to use)
+7. Do you need to use database:
+   - if YES - correct liquibase scripts in `resources/config/liquibase`
+   - if NO - remove liquibase scripts, config `XmDatabaseConfiguration` and commons `xm-commons-migration-db`
+8. Check all `com.icthh.xm.commons` libraries in `build.gradle` and remove redundant. try to keep minimum set of commons.
```

Comment:
seems we do not need this after template was applied actually,

Reply:
removed